src/versions/default_versions.ts

```diff
@@ -54,23 +54,23 @@ export const DefaultLinuxVersion: Record<string, Record<number, string> | undefi
     20: "7.0.0-2",
   },
   llvm: {
-    22: "17.0.6-ubuntu-22.04",
-    20: "17.0.6-ubuntu-22.04",
-    18: "15.0.6-ubuntu-18.04",
+    22: "18.1.1-ubuntu-22.04",
+    20: "18.1.1-ubuntu-20.04",
```

Comment:
These prefixes relate to the prefixes the LLVM uses in their binaries (listed in `llvm_url`). These should be all `ubuntu-22.04` if that's what LLVM used

Reply:
thanks fixed. thought they were referring the deb packages versions.